src/console/components/cz-console-2/main.js

```diff
@@ -0,0 +1,338 @@
+    button.textContent = 'Dir';
+    button.onclick = () => {
+      console.dir(document.location);
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getDirXML(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'DirXML';
+    button.onclick = () => {
+      console.dirxml(document.location);
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getError(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'Error';
+    button.onclick = () => {
+      console.error('This is an error message!');
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getGroup(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'Group';
+    button.onclick = () => {
+      console.group('Level 2');
+      console.log('Level 2');
+      console.group('Level 3');
+      console.log('Level 3');
+      console.log('More of level 3');
+      console.groupEnd();
+      console.log('Back to level 2');
+      console.groupEnd();
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getGroupCollapsed(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'GroupCollapsed';
+    button.onclick = () => {
+      console.groupCollapsed('Level 2');
+      console.log('Level 2');
+      console.log('You had to displayed it right?');
+      console.group('Level 3');
+      console.log('Level 3');
+      console.log('More of level 3');
+      console.log('You had to displayed it right?');
+      console.groupEnd();
+      console.log('Back to level 2');
+      console.groupEnd();
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getInfo(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'Info';
+    button.onclick = () => {
+      console.info('This is an info message');
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getLog(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'Log';
+    button.onclick = () => {
+      console.log(document.body);
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getProfile(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'Profile';
+    button.onclick = () => {
+      console.profile();
+    };
+    shadow.appendChild(button);
+  }
+  /**
+   *
+   * @param {shadowroot} shadow
+   */
+  getProfileEnd(shadow) {
+    const button = document.createElement('button');
+    button.textContent = 'ProfileEnd';
+    button.onclick = () => {
+      console.profileEnd();
+    };
+    shadow.appendChild(button);
+  }
```

Comment:
Since console.profileEnd does nothing without console.profile you can perhaps merge these two methods into one and call the profileEnd once the profile loop ends. (yeah add a loop, interval or something to show how a profile happens)